azure-pipelines.yml

```diff
@@ -246,7 +246,7 @@ extends:
                   androidAppHostingSdkGitPath: AndroidAppHostingSdk
 
           - job: E2ETestIOS
-            displayName: 'E2E Test - IOS'
+            displayName: 'E2E Test - IOS - Plan A'
```

Comment:
I'm not a huge fan of Plan A and Plan B nomenclature, but drawing a blank on alternatives for now.

Reply:
This blank cannot be updated dynamically so I will use plan A/B to indicate which iOS test plan (there is a plan file) is used. Once I figure out how to dynamically do that as web hub sdk does, I can update here to use alphabetical range.

Reply:
Sounds good, thanks